lib/ood_core/job/adapters/pbspro.rb

```diff
@@ -44,15 +47,22 @@ class Batch
           # @return [Pathname, nil] path to pbs executables
           attr_reader :exec
```

Comment:
`exec` is a Kernel method - lets fix this to be `bin` like the other adapters. I guess it is written to mirror the configuration hash which is:

https://osc.github.io/ood-documentation/master/installation/resource-manager/pbspro.html?highlight=pbspro

and contains `exec: "/path/to/pbspro"`. Is there a PBSPro reason why this is the case? Perhaps we should change this by default to be bin, and do something like this in the `build_pbspro` factory method

```
bin = c.fetch(:bin) { c.fetch(:exec, nil) }
```

This will also make sense then so that the naming is similar (bin, custom_bin vs exec, custom_bin). Then of course the `PBSPro::Batch` would be updated so that `attr_reader :exec` is `attr_reader :bin`.